docs/developer/other/RELEASE.md

```diff
@@ -35,6 +35,30 @@ We encourage performing releases directly from a clone of the official repositor
 
 This means when testing on `uPortal-start` for the release, you should use the `apereo` repository but configure the `uPortalVersion` to be the SNAPSHOT version that you'll build in the following steps.
 
+## Send Pre-Release Notice to Community
+For any non-snapshot release, email a notice to `uportal-dev` a couple of working days prior to cutting the release.
+- Request any in-progress Pull Requests be merged by a certain date/time
+- Request adopters test the tip of `master`
+
+## Review Dependencies
+
+Before releasing uPortal and updating uPortal-start with the latest uPortal version, review the dependencies and update dependencies where appropriate.
+
+Assuming adopters use the community build of uPortal, there are three primary sets of dependencies to be considered:
+- uPortal
+- uPortal-start (community version)
+- uPortal-start (customized)
+
+Ideally, dependencies in all of the above areas should be at the latest versions.  In reality, dependencies may lag, and be out of sync with each other.
+
+### Specific Dependencies
+
+- `nodejsVersion` - uPortal and uPortal-start use NodeJS for building artifacts and Gradle tasks. They are not required to be in sync.
```

Comment:
Ideally they should be on the same major version.
But minor and patch versions don't need to match. 👍